### DISCUSSION TITLE
How to set TEMPORAL_HISTORY_RETENTION_IN_DAYS and minIO volume size for helm chart

### DISCUSSION THREAD

**Discussion by tgquan67** [airbytehq/airbyte#35323] [Connector Questions]
I have seen articles about setting `TEMPORAL_HISTORY_RETENTION_IN_DAYS` env var, however I looked into the [helm chart's values.yaml file](https://github.com/airbytehq/airbyte-platform/blob/main/charts/airbyte/values.yaml) or [temporal chart's](https://github.com/airbytehq/airbyte-platform/blob/main/charts/airbyte-temporal/values.yaml) and found nothing resembles the variable. Which value should I set to achieve the effect similar to that env var?   
And on that topic, is my understanding correct that the var will have the same effect as manually cleaning up the log as instructed in t[his comment](https://discuss.airbyte.io/t/how-to-clean-up-airbyte-minio-and-whats-stored-in-minio/3101/5)?   
In the case that I cannot set that env var, what is the way to set the size for the log storage volume (and other volumes) when I deploy using helm chart?

**Comment by cernadasjuan** [comment]
same issue here! I don't find anything

**Comment by thatduy** [comment]
Are there any update about TEMPORAL_HISTORY_RETENTION_IN_DAYS?

[a comment by an automated account is not shown]
